tests/system/Commands/Utilities/Routes/FilterCollectorTest.php

```diff
@@ -36,7 +36,6 @@ public function testGet(): void
             'before' => [
             ],
             'after' => [
-                'toolbar',
```

Comment:
Is it intentional that Filter collection doesn't include `$required` filters? That seems counterintuitive.

Reply:
Required filters are always surely executed. Do you really need to show them in the commands?